iam_check/parameters.py

```diff
@@ -46,4 +56,17 @@ def validate_finding_types(finding_types):
 		if finding_type not in ['ERROR', 'SECURITY_WARNING', 'SUGGESTION', 'WARNING', 'NONE']:
 			raise ArgumentTypeError(f"Invalid finding type: {finding_type}.")
 
-	return finding_types
\ No newline at end of file
+	return finding_types
+
+def validate_finding_codes(finding_codes):
+	if finding_codes is None:
+		return finding_codes
+
+	finding_codes = [finding_code.strip() for finding_code in finding_codes]
+	finding_codes = [finding_code.upper() for finding_code in finding_codes]
+
+	# for finding_code in finding_codes:
+	# 	if finding_code not in ['ERROR', 'SECURITY_WARNING', 'SUGGESTION', 'WARNING', 'NONE']:
+	# 		raise ArgumentTypeError(f"Invalid finding code: {finding_code}.")
```

Comment:
is there any way to pull all possible finding codes? otherwise validation is just "does it split"